<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewDockedWidgets</class>
 <widget class="QMainWindow" name="PreviewDockedWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QDockWidget" name="iv_dock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="iv_dock_content">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="iv_layout">
       <item>
        <layout class="QHBoxLayout" name="iv_toolbar_layout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QToolButton" name="iv_zoom_button">
           <property name="toolTip">
            <string>Move the instrument</string>
           </property>
           <property name="text">
            <string>Zoom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="iv_edit_button">
           <property name="toolTip">
            <string>Select and edit shapes</string>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="iv_rect_select_button">
           <property name="toolTip">
            <string>Draw a rectangle</string>
           </property>
           <property name="text">
            <string>Rect. Select</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="iv_toolbar_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="iv_placeholder" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="rs_dock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="rs_dock_content">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="rs_plot_layout">
       <item>
        <layout class="QHBoxLayout" name="rs_toolbar_layout">
         <item>
          <widget class="QToolButton" name="rs_ads_export_button">
           <property name="toolTip">
            <string>Export the summed workspace to the workspaces list</string>
           </property>
           <property name="text">
            <string>Export summed workspace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="rs_edit_button">
           <property name="toolTip">
            <string>Edit a rectangular region of interest</string>
           </property>
           <property name="text">
            <string>Edit ROI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="rs_rect_select_button">
           <property name="toolTip">
            <string>Select a rectangular region of interest</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="popupMode">
            <enum>QToolButton::MenuButtonPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="rs_toolbar_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="region_selector" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="lp_dock">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="lp_dock_content">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="lp_layout">
       <item>
        <widget class="QToolButton" name="lp_ads_export_button">
         <property name="toolTip">
          <string>Export the reduced workspace to the workspaces list</string>
         </property>
         <property name="text">
          <string>Export reduced workspace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MantidQt::MantidWidgets::QtPlotView" name="line_plot" native="true">
         <property name="toolTip">
          <string>Reduced data. Use right-click for plot options</string>
         </property>
         <property name="toolTipDuration">
          <number>2000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::QtPlotView</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Plotting/PlotWidget/QtPlotView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
